app/controllers/concerns/blacklight/suggest/suggest_helper.rb

```diff
@@ -0,0 +1,32 @@
+module Blacklight
+  module Suggest
+    module SuggestHelper
```

Comment:
It could be. I wanted to separate the action methods from the concern of methods that "do more stuff"... Prefer to put it all in one place?